bundle/Resources/config/kernel.yml

```diff
@@ -2,3 +2,8 @@ system:
     default:
         fielddefinition_edit_templates:
             - { template: "EzSystemsRepositoryFormsBundle:ContentType:field_types.html.twig", priority: 0 }
+        location_view:
+            _platformui_content_name:
+                content_name:
```

Comment:
@lolautruche ...but default_view doesn't work (InvalidConfigurationException, Unrecognized option "default_view" under "ezpublish.system.default") and since I'm using `ez_content:viewLocation`, it makes sense to me :)